packages/editor/src/editor/palette/PaletteElement.tsx

```diff
@@ -0,0 +1,10 @@
+import { Flex } from '@axonivy/ui-components';
+import type { ReactNode } from 'react';
+import './PaletteElement.css';
+
+export const PaletteElement = ({ text, children }: { text: string; children: ReactNode }) => (
+  <Flex direction='column' alignItems='center' gap={1} className='palette-element'>
+    <span className='category-label'>{text}</span>
+    {children}
+  </Flex>
+);
```

Comment:
please create a new component (I would call it `PaletteButton`) and also use it in the PalettePopover as button.